lib/msal-node/docs/errors.md

```diff
@@ -0,0 +1,171 @@
+# Errors
+
+***
+
+**[ClientAuthErrors](#Clientautherrors)**
+
+1.[endpoints_resolution_error](#endpoints_resolution_error)
+
+2.[openid_config_error](#openid_config_error)
+
+3.[invalid_assertion](#invalid_assertion)
+
+4.[request_cannot_be_made](#request_cannot_be_made)
+
+5.[invalid_client_credential](#invalid_client_credential)
+
+**[ClientConfigurationErrors](#ClientConfigurationErrors)**
+
+1.[empty_url_error](#empty_url_error)
+
+2.[url_parse_error](#url_parse_error)
+
+**[Other](#other)**
+
+## ClientAuthErrors
+
+### Endpoints_resolution_error
+
+**Error Message**: Error: could not resolve endpoints. Please check network and try again.
+
+This error is thrown under the following scenarios:
+
+1. When a wrong authority endpoint is provided. Details: ClientAuthError: `openid_config_error`: Could not retrieve endpoints. Check your authority and verify the .well-known/openid-configuration endpoint returns the required endpoints. 
+
+Ensure that the authority provided matches the authority in your azure app registration.
+```javascript
+const config = {
+    auth: {
+        authority: "https://login.microsoftonline.com/<provide-authority-that-matches-your-app-registration>"
+        // ...   
+    }
+}
+```
+
+2. When there is no internet connectivity. Details: ClientConfigurationError: `untrusted_authority`: The provided authority is not a trusted authority. Please include this authority in the knownAuthorities config parameter. It is also thrown when no authority is appended to the authority endpoint.
```

Comment:
No internet connectivity is different from `untrusted_authority`, this should be 2 separate items

Reply:
+1. Node should detect network connectivity errors separately